src/modules/navigator/mission_block.cpp

```diff
@@ -277,9 +277,15 @@ MissionBlock::is_mission_item_reached()
 			_time_wp_reached = now;
 
 		} else {
-			/*normal mission items */
 
-			float mission_acceptance_radius = _navigator->get_acceptance_radius();
+			float acceptance_radius = _navigator->get_acceptance_radius();
+
+			// We use the acceptance radius of the mission item if it has been set (not NAN)
+			// but only for multicopter.
+			if (_navigator->get_vstatus()->vehicle_type == vehicle_status_s::VEHICLE_TYPE_ROTARY_WING
+			    && PX4_ISFINITE(_mission_item.acceptance_radius) && _mission_item.acceptance_radius > 0.0f) {
```

Comment:
Is a comparison >0.0f safer than >FLT_EPSILON?

Reply:
Right, let me change that.